.gitignore

```diff
@@ -33,6 +33,8 @@ __pycache__/
 /scanpy.egg-info/
 /*-env/
 /env-*/
+# This is necessary to make flit generate a setup.py
+/setup.py
```

Comment:
```suggestion
```

I removed this from the `.gitignore` and everything seemed to work fine. (`flit install`, `flit install -s`, `flit install --pth-file`, `flit build`).

Could this get removed? Alternatively, could you provide an example where this causes errors?

Reply:
I clarified the reasoning.